tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -101,10 +101,10 @@ Docker run ps password check
     \   Should Not Contain  @{command}[4]  %{TEST_PASSWORD}
 
 Docker run immediate exit
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run busybox
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run ${busybox}
     Should Be Equal As Integers  ${rc}  0
     Should Be Empty  ${output}
 
```

Comment:
Commenting here since I can't seem to add a comment against line 112:
```
docker %{VCH-PARAMS} pull busybox
```
should use `${busybox}`